pkgs/development/ocaml-modules/ocaml-libvirt/default.nix

```diff
@@ -19,9 +19,8 @@ stdenv.mkDerivation rec {
 
   createFindlibDestdir = true;
 
-  buildPhase = "make all opt CPPFLAGS=-Wno-error";
-
-  installPhase = "make install-opt";
+  buildFlags = "all opt CPPFLAGS=-Wno-error";
```

Comment:
Let's make this a list while we're at it?!

Reply:
Done.